### ISSUE TITLE
Crash Report 12.51

### ISSUE THREAD

**Issue by brunowoelke** [opentibiabr/otservbr-global-archived#1878]
otserver    | Thread 1 "otbr" received signal SIGSEGV, Segmentation fault.
otserver    | 0x0000563dc9d9ec47 in Player::postRemoveNotification(Thing*, Cylinder const*, int, cylinderlink_t) ()
otserver    | #0  0x0000563dc9d9ec47 in Player::postRemoveNotification(Thing*, Cylinder const*, int, cylinderlink_t) ()
otserver    | #1  0x0000563dc9eb2f7e in Game::internalRemoveItem(Item*, int, bool, unsigned int) [clone .constprop.797] ()
otserver    | #2  0x0000563dc9e1f739 in Game::internalRemoveItems(std::vector<Item*, std::allocator<Item*> >, unsigned int, bool) [clone .constprop.443] ()
otserver    | #3  0x0000563dc9d9509c in Player::removeItemOfType(unsigned short, unsigned int, int, bool) const ()
otserver    | #4  0x0000563dc9cd2df5 in Player::stowContainer(Item*, unsigned int, bool) ()
otserver    | #5  0x0000563dc9dbf6a1 in ProtocolGame::parsePacket(NetworkMessage&) ()
otserver    | #6  0x0000563dc9e01c4a in Connection::parsePacket(boost::system::error_code const&) ()
otserver    | #7  0x0000563dc9e05f5c in boost::asio::detail::reactive_socket_recv_op<boost::asio::mutable_buffers_1, boost::asio::detail::read_op<boost::asio::basic_stream_socket<boost::asio::ip::tcp, boost::asio::stream_socket_service<boost::asio::ip::tcp> >, boost::asio::mutable_buffers_1, boost::asio::detail::transfer_all_t, std::_Bind<void (Connection::*(std::shared_ptr<Connection>, std::_Placeholder<1>))(boost::system::error_code const&)> > >::do_complete(boost::asio::detail::task_io_service*, boost::asio::detail::task_io_service_operation*, boost::system::error_code const&, unsigned long) ()
otserver    | #8  0x0000563dc9ce4937 in main ()

**Comment by heberholanda** [comment]
Update your source.

**Comment by oalbsilva** [comment]
Do you have more details?

**Comment by brunowoelke** [comment]
Sorry, no.
But it happened only once in 4 days.

**Comment by heberholanda** [comment]
There was actually a bug about this, but it has already been fixed. You can test it and if it occurs again open another issue.
